src/cow-react/api/gnosisSafe/index.ts

```diff
@@ -19,35 +20,41 @@ const CHAIN_SHORT_NAME: Partial<Record<number, string>> = {
   [ChainId.GOERLI]: 'gor', // https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-5.json
 }
 
-const SAFE_TRANSACTION_SERVICE_CACHE: Partial<Record<number, SafeServiceClient | null>> = {}
+const SAFE_TRANSACTION_SERVICE_CACHE: Partial<Record<number, SafeApiKit | null>> = {}
 
-function _getClient(chainId: number, library: Web3Provider): SafeServiceClient | null {
+const txServiceUrl = 'https://safe-transaction-goerli.safe.global'
```

Comment:
I guess we should have a map for all networks

Reply:
This was not needed. We already have the URL by network, it was just a test. 

I just removed it and use the right url